CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 5.0.0-beta.0 (2021-07-08)
+
+- Initial beta release.
+- Added support for `@angular/core@^12`. [#53](https://github.com/blackbaud/skyux-autonumeric/pull/53)
```

Comment:
I don't see this change in this PR. I see an updated to `@skyux/core` but not `@angular/core`. Am I just overlooking it?

Reply:
We made the code change in the initial commit to the branch, so there unfortunately isn't a PR that shows the change. So, I just attributed it to the most recent PR after the change. I'm trying to avoid that going forward, but there will be a few more like this.